.github/workflows/release.yaml

```diff
@@ -28,14 +34,13 @@ jobs:
           key: cache-${{ env.cache-version }}-go-${{ env.go-version }}-${{ hashFiles('v2/Makefile') }}
       - run: make setup
         if: steps.cache-tools.outputs.cache-hit != 'true'
-      - run: make image
       - name: Login to ghcr.io
         run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
-      - name: Push to ghcr.io
+      - run: make build
```

Comment:
The go binary that will be output in this step will be built using the OS architecture of the GHA build environment (linux/amd64). This is probably not what you want as the ARM image you are building in the L43 step will end up containing the AMD64 image and therefore not run.